Simplenote/src/androidTest/java/com/automattic/simplenote/integration/AddTagActivityTest.kt

```diff
@@ -0,0 +1,136 @@
+package com.automattic.simplenote.integration
+
+import android.app.Activity
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.replaceText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.ActivityResultMatchers.hasResultCode
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.MediumTest
+import androidx.test.runner.AndroidJUnit4
+import com.automattic.simplenote.AddTagActivity
+import com.automattic.simplenote.BaseUITest
+import com.automattic.simplenote.R
+import com.automattic.simplenote.utils.hasTextInputLayoutErrorText
+import org.hamcrest.CoreMatchers.not
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@MediumTest
+class AddActivityTest : BaseUITest() {
+
+    @Test
+    fun addNewValidTag() {
+        Assert.assertEquals(tagsBucket.count(), 0)
```

Comment:
Minor (🔍): Consider moving the `Assert.` bit to the imports sections. This will then make every `assertEquals` like less involved. Wdyt?

Reply:
I agree. I moved it `Assert` to imports section.